src/main/kotlin/org/rust/ide/intentions/UnwrapToTryIntention.kt

```diff
@@ -0,0 +1,31 @@
+package org.rust.ide.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.RsMethodCallExpr
+import org.rust.lang.core.psi.RsPsiFactory
+import org.rust.lang.core.psi.RsTryExpr
+import org.rust.lang.core.psi.util.parentOfType
+
+class UnwrapToTryIntention : RsElementBaseIntentionAction<RsMethodCallExpr>() {
+    override fun getText() = "Replace .unwrap() with try"
+    override fun getFamilyName() = text
+
+    override fun findApplicableContext(project: Project, editor: Editor, element: PsiElement): RsMethodCallExpr? {
+        val methodCall = element
+            .parentOfType<RsMethodCallExpr>(strict = false) ?: return null
+        if (methodCall.identifier.textMatches("unwrap") &&
+            methodCall.typeArgumentList == null &&
+            methodCall.valueArgumentList.exprList.isEmpty()) {
+            return methodCall
+        }
+        return null
+    }
+
+    override fun invoke(project: Project, editor: Editor, ctx: RsMethodCallExpr) {
+        val newPsiElement = RsPsiFactory(project).createExpression("a?")
+        (newPsiElement as? RsTryExpr)?.expr?.replace(ctx.expr) ?: return
```

Comment:
In this cases, I think it's better to through error (which contains the troublesome expression text) than to silently do nothing. 

If the user sees an exception and a stack trace, she is more likely to send a bug report, and we want bug reports to fix the bugs. 

So, lets add `createTryExpression(inner: RsExpr): RsTryExpr` to the `RsPsiFactory`.